### ISSUE TITLE
EXPO eas build FAILED on run-ios

### ISSUE THREAD

**Issue by Speakergr** [gev2002/react-native-vision-camera-text-recognition#1]
Hi! 
When build with expo  -> Conflict with "@react-native-firebase/storage" and "react-native-vision-camera-text-recognition" 

https://prnt.sc/kJf553jcK5XC

**package.json:**
"@react-native-firebase/analytics": "^20.0.0",
"@react-native-firebase/app": "^20.0.0",
"@react-native-firebase/auth": "^20.0.0",
"@react-native-firebase/database": "^20.0.0",
"@react-native-firebase/firestore": "^20.0.0",
"react-native-vision-camera": "^4.0.5",
"react-native-vision-camera-text-recognition": "^1.0.1",
 "react-native-worklets-core": "^1.3.3",

**Comment by gev2002** [comment]
Are you tried to clean cache and clean build folders and rebuild your app?

**Comment by Speakergr** [comment]
Yes, I tried ... from expo and by command eas build -p ios --clear-cache

**Comment by Speakergr** [comment]
any way change version for " GoogleMLKit/TextRecognition (= 2.2.0)" to more upper?
p.s. sorry for my English

**Comment by gev2002** [comment]
Yes, I'll do it tomorrow, I should change logic.

**Comment by Speakergr** [comment]
thanks... a lot!!!
